packages-content-model/roosterjs-content-model-types/lib/parameter/Position.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Represent a position in DOM tree by the container and its offset index
+ */
+export interface Position {
```

Comment:
Sounds good, removed this interface as it will not be needed and we can just retrieve the position using the range and whether the selecion is reverted or not.